pkg/controlplane/reconcilers/helpers_test.go

```diff
@@ -41,3 +48,51 @@ func makeEndpoints(name string, ips []string, ports []corev1.EndpointPort) *core
 	}
 	return endpoints
 }
+
+func verifyCreatesAndUpdates(fakeClient *fake.Clientset, expectedCreates, expectedUpdates []runtime.Object) error {
+	errors := []error{}
```

Comment:
just a comment, maybe you already know, golang has this `t.Helper()` method for these kind of functions